src/Common/test/uk/ac/ox/zoo/seeg/abraid/mp/common/dao/ModelRunDaoTest.java

```diff
@@ -68,9 +68,12 @@ public void saveAndReloadModelRunByName() {
         DateTime batchStartDate = DateTime.now().minusHours(6).minusDays(1);
         DateTime batchEndDate = DateTime.now().minusHours(6);
         DateTime batchingCompletionDate = DateTime.now().plusHours(2);
+        DateTime occurrenceDataRangeStartDate = DateTime.now().plusHours(9);
+        DateTime occurrenceDataRangeEndDate = DateTime.now().plusHours(8);
```

Comment:
It isn't so critical in this test situation, but here start date > end date. Could this (or should this) have some validation or constraint on it somewhere?